src/js/components/user-management/signup.test.js

```diff
@@ -19,7 +19,7 @@ describe('Signup Component', () => {
     const { baseElement } = render(
       <MemoryRouter>
         <Provider store={store}>
-          <Signup location={{ state: { from: '' } }} />
+          <Signup location={{ state: { from: '' }, match: { params: { campaign: '' } } }} />
```

Comment:
@tranchitella the location is a misleading copy&paste remnant from the login component test
```suggestion
          <Signup match={{ params: { campaign: '' } }} />
```
should work & allow the access to the match without conditions

Reply:
Thanks!